f8a_report/helpers/report_helper.py

```diff
@@ -64,32 +64,40 @@ def __init__(self):
 
         self.emr_api = os.getenv('EMR_API', 'http://f8a-emr-deployment:6006')
 
+    def cleanup_tables(self, table_name, column_name, num_days):
+        """Cleanup tables on a periodic basis."""
+        # Query to delete data
+        query = sql.SQL('DELETE FROM {0} WHERE {1} <= NOW() - interval \'%s day\';').format(
```

Comment:
Please use multi line/raw strings to avoid using escape chars, thats one of purpose of that.
```suggestion
        query = sql.SQL("""DELETE FROM {0} WHERE {1} <= NOW() - interval '%s day';""").format(
```

Reply:
On it.